hi-ml/testhiml/testhiml/test_checkpoints.py

```diff
@@ -153,9 +157,8 @@ def test_custom_checkpoint_for_test(tmp_path: Path) -> None:
     does_not_exist = Path("does_not_exist")
     with mock.patch("health_ml.configs.hello_world.HelloWorld.get_checkpoint_to_test") as mock2:
         mock2.return_value = does_not_exist
-        with pytest.raises(FileNotFoundError) as ex:
+        with pytest.raises(FileNotFoundError):
             checkpoint_handler.get_checkpoint_to_test()
-        assert str(does_not_exist) in str(ex)
```

Comment:
why did this get deleted? did the log change?

Reply:
Yes, the message changed, the name of the file is no longer in the error message. The filename is in other log messages though, so it can be recovered easily. I'm adding a `match` to check for the right exception back in, thanks for spotting that.